File: Cross-Cutting/Cross-Cutting-Artifacts/src/main/java/co/com/touresbalon/foundation/crosscutting/exceptions/BusinessException.java
package co.com.touresbalon.foundation.crosscutting.exceptions;

import javax.ejb.ApplicationException;

/**
 * Created by garciniegas on 28/08/2015.
 */

@ApplicationException(rollback = true)
public class BusinessException extends Exception {

    public BusinessException() {
    }

    public BusinessException(String message) {
        super(message);
    }

    public BusinessException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: Microservices-Module/Customers/Cross-Cutting/microsoft/co/com/touresbalon/foundation/crosscutting/exception/BusinessException.cs
using System;

namespace Cross_Cutting.microsoft.co.com.touresbalon.foundation.crosscutting.exception
{
    public class BusinessException : Exception
    {
        public BusinessException(string message) : base(message) { }

        public BusinessException(string message, Exception innerException) { }
    }
}
